src/modules/postComment.js

```diff
@@ -0,0 +1,21 @@
+import addComment from './addComment.js';
+
+export default ({ id, username, comment }) => {
+  const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/mU67F6D8b0eq3yf1Ab3T/comments';
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+      username,
+      comment,
+    }),
+  }).then(() => {
+    addComment({
+      name: username,
+      comment,
+    });
+  });
+};
```

Comment:
- [ ] As requested in the project requirements, you should use async and await with the fetch method to send and receive data. Please add async and await to this function.